### DISCUSSION TITLE
Shape recognition

### DISCUSSION THREAD

**Discussion by wikyburns** [cnr-isti-vclab/PyMeshLab#66] [Ideas]
Hi guys, I have so many time trying to found something to detect a shape on a 3D file like ply, obj or stl.

Someone has idea about that?

That can solve a lot of real problems.

**Comment by alemuntoni** [comment]
What exactly do you mean with "detect a shape"? What should be the input and what the output?

**Comment by wikyburns** [comment]
> What exactly do you mean with "detect a shape"? What should be the input and what the output?

For example if i have an input of an .obj file, with a plantar foot, and other objects arround this but I only need the plantar foot. What I would like to have is the output of only the plantar foot. Ignoring the objects arround.
